src/main/frontend/handler/export.cljs

```diff
@@ -430,25 +394,6 @@
           (.setAttribute anchor "download" (.-name zipfile))
           (.click anchor))))))
 
-(defn export-page-as-markdown!
```

Comment:
Both `export-page-as-markdown` and `export-page-as-opml!` are used in https://github.com/logseq/logseq/blob/master/src/main/frontend/components/export.cljs#L46, is it ignored because of the added `:clj-kondo/ignore`? If so, is it possible that there're other unused vars that are actually used?

Reply:
ignore should only effect clj-kondo linting and not its analysis phase, which is what carve uses. `export-page` was no longer used after https://github.com/logseq/logseq/commit/40a5cecaf3dc92798372903e8660ad601998f417